WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -559,26 +559,38 @@ private extension ShippingLabelFormViewController {
 // MARK: - Top Banner
 //
 private extension ShippingLabelFormViewController {
-    /// Creates a Top Banner View containing the EU Shipping Notice.
+    /// Present a Top Banner View containing the EU Shipping Notice.
     ///
     func showTopBannerView() {
-        guard viewModel.shouldDisplayShippingNotice else {
-            return
+        viewModel.shouldDisplayEUShippingNotice { [weak self] shouldDisplay in
+            guard let self = self, shouldDisplay else {
```

Comment:
nit: You can simplify `guard let` here:

```suggestion
            guard let self, shouldDisplay else {
```

Reply:
Oh, nice! I will remind myself to avoid this in upcoming changes! Thanks!